packages/arb-token-bridge-ui/package.json

```diff
@@ -43,13 +43,15 @@
     "swr": "^2.1.2",
     "tailwind-merge": "^0.9.0",
     "use-query-params": "^2.1.1",
+    "viem": "^1.16.6",
```

Comment:
I understand we want to move to viem at some point, but I'm not a big fan as doing it as part of this PR. Did we check how much it adds to the bundle?

Reply:
Yeah I feel like the PR covers 2 completely separate issues

Reply:
yea i'll revert to sdk~ good point